src/services/infoService.ts

```diff
@@ -0,0 +1,164 @@
+// services/infoService.ts
+import { Request, Response } from 'express';
+import { StatusCode } from '../enum/AppConst';
+import Info from '../models/Info'; // Import Info and InfoAttributes
+
+const getInfo = async (req: Request, res: Response) => {
+  try {
+    const info = await Info.findOne(); // Assuming you only have one set of info in the database
```

Comment:
e chưa rõ ý đồ chỗ này lắm thường get data của 1 đối tượng thì sẽ lấy theo id, còn đối tượng là duy nhất của system thì mới getOne (vd: get config hệ thống)

Reply:
info này là dùng quản lý toàn bộ thông tin cơ bản của cty á mà :)) nó chỉ có một
tên, địa chỉ, sdt,.....